src/ray/gcs/gcs_client/accessor.h

```diff
@@ -885,6 +885,23 @@ class InternalKVAccessor {
   GcsClient *client_impl_;
 };
 
+class RuntimeEnvAccessor {
+ public:
+  RuntimeEnvAccessor() = default;
```

Comment:
wondering why we need this

Reply:
All other accessors have it. I think it's for mocks: https://github.com/ray-project/ray/blob/ae3e87cb0bb51fecde718f1e9fbc4ced488c8986/src/mock/ray/gcs/gcs_client/accessor.h#L232